frontend/settings/OBSBasicSettings.cpp

```diff
@@ -3253,22 +3191,27 @@ static void WriteJsonData(OBSPropertiesView *view, const char *path)
 	}
 }
 
-static void SaveTrackIndex(config_t *config, const char *section, const char *name, QAbstractButton *check1,
-			   QAbstractButton *check2, QAbstractButton *check3, QAbstractButton *check4,
-			   QAbstractButton *check5, QAbstractButton *check6)
+static int RadioToIdx(std::vector<QPointer<QRadioButton>> values)
```

Comment:
Use anonymous namespace for free-standing functions like these and put them at the top of the file, so it's visible at a glance what kind of "utility" functionality is available throughout.

Reply:
I'm not very fluent in C++, but as soon as I understand, making it `static` already makes it private to the current file? Or do I have to manually wrap it into `namespace {}`? Primarily, I was doing everything the same way as it was done in the code around. Please let me know how to fix it because I'm very new to C++.

Reply:
Global `static` functions were originally deprecated in C++11, but the standards body reverted that because they figured out that they are not able to truly remove it from the language.

But it has been part of the C++ Core guidelines to use anonymous namespaces: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf22-use-an-unnamed-anonymous-namespace-for-all-internalnon-exported-entities